### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, lstat '/Users/markrodgers/Desktop/Funeralcare/...

### ISSUE THREAD

**Issue by marr975** [tomkadwill/atom-cucumber-autocomplete#19]
[Enter steps to reproduce below:]
1. While working with .feature file type either 'Given', 'When' or 'Then' then type a space.
2. Type another word then type a space.

The following error occurs.

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [cucumber-autocomplete](https://github.com/tomkadwill/atom-cucumber-autocomplete) package, v0.6.3
### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat '/Users/markrodgers/Desktop/Funeralcare/fnc-development-environment/features'

```
At ATOM_SHELL_ASAR.js:0

Error: ENOENT: no such file or directory, lstat '/Users/markrodgers/Desktop/Funeralcare/fnc-development-environment/features'
    at Error (native)
    at fs.lstatSync (fs.js:887:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:229:16)
    at Object.scanFeaturesDir (/Users/markrodgers/.atom/packages/cucumber-autocomplete/lib/provider.js:46:13)
    at Object.getCompletions (/Users/markrodgers/.atom/packages/cucumber-autocomplete/lib/provider.js:31:19)
    at Object.getSuggestions (/Users/markrodgers/.atom/packages/cucumber-autocomplete/lib/provider.js:17:17)
    at /Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:310:56
    at Array.forEach (native)
    at AutocompleteManager.module.exports.AutocompleteManager.getSuggestionsFromProviders (/Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:289:17)
    at AutocompleteManager.getSuggestionsFromProviders (/Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:3:61)
    at AutocompleteManager.module.exports.AutocompleteManager.findSuggestions (/Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:276:19)
    at /Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:3:61
```
### Commands

```
     -0:14 core:backspace (atom-text-editor.editor.is-focused)
     -0:04.2.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:01.3.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "language-rubymotion"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
autocomplete-motion, v2.1.0
cucumber, v0.5.0
cucumber-autocomplete, v0.6.3
language-r, v0.4.1
language-rspec, v0.5.0
language-scala, v1.1.6

# Dev
No dev packages
```

**Comment by tomkadwill** [comment]
Hi @marr975, I've added a fix for this in v0.7.1. Please re-test re-open this issue if the problem still occurs in v0.7.1

**Comment by marr975** [comment]
Tested - Resolved.
